Client/Frontend/TabBar/SearchViewController.swift

```diff
@@ -151,7 +151,8 @@ extension SearchViewController: UITableViewDelegate {
             let suggestion = searchSuggestions[indexPath.row]
 
             // Assume that only the default search engine can provide search suggestions.
-            url = searchEngines?.defaultEngine.searchURLForQuery(suggestion)
+//            url = searchEngines?.defaultEngine.searchURLForQuery(suggestion)
+            url = URIFixup().getURL(searchSuggestions[indexPath.row])
```

Comment:
This will break suggestions that aren't URLs. I think you'll want to add this line above the existing line, then do the following <code>searchURLForQuery</code> only if <code>url</code> is still <code>nil</code>.